docs/modules/ROOT/pages/reference/extensions/mybatis.adoc

```diff
@@ -34,3 +38,25 @@ Please refer to the above links for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Additional Camel Quarkus configuration
+
+Please refer to https://quarkiverse.github.io/quarkiverse-docs/quarkus-mybatis/dev/index.html[Quarkus MyBatis] for configuration. It must enable the following options
+
+[source, properties]
+----
+quarkus.mybatis.xmlconfig.enable=true
+quarkus.mybatis.xmlconfig.path=SqlMapConfig.xml
+----
+TIP: `quarkus.mybatis.xmlconfig.path` must be the same with `configurationUri` param in the mybatis endpoint.
+
+If you want to run in the native mode, you have to set these properties in pom.xml currently.
+
+[source, xml]
+----
+<properties>
+    <quarkus.package.type>native</quarkus.package.type>
+    <quarkus.native.additional-build-args>--report-unsupported-elements-at-runtime</quarkus.native.additional-build-args>
```

Comment:
Could you please explain why is this required? We generally do not want any `additional-build-args`.

Reply:
OK, now I see there is https://github.com/quarkiverse/quarkus-mybatis/issues/10 . I'd prefer not merging this unless https://github.com/quarkiverse/quarkus-mybatis/issues/10 is fixed, or unless there is a clear idea how it can be fixed.